ci

```diff
@@ -448,6 +448,25 @@ case "$1" in
     refresh_fstar_hints
     ;;
 
+  fstar-docs-nightly)
+    # Create the docs associated with FStar
+    if [ ! -d ulib ]; then
+      echo "I don't seem to be in the right directory, bailing"
+      exit 1
+    fi
+
+    # First - get fstar built
+    git clean -ffdx
```

Comment:
why do you clean and reset the repository? isn't it something that VSTS is meant to do for you?